docs/uab_cloud/tutorial/volumes.md

```diff
@@ -198,3 +198,30 @@ Click "Delete Volume" again to delete the volume permanently.
 
     It will not be possible to delete a volume if it has an associated [volume snapshot](../snapshots.md). The snapshot will need to be deleted first.
 <!-- markdownlint-enable MD046 -->
+
+## Increasing a Persistent Volume
+
+In this tutorial, we'll guide you through the process of resizing your volumes directly from the dashboard. Whether you need more storage or want to optimize your resource allocation, you can achieve either with just a few clicks. Follow the steps below to update your volume size.
```

Comment:
I don't think we need a prose-style introduction. If you can, try to mirror the style of the existing text. If you do include an introduction, try to focus on the practical effects and benefits/drawbacks, and any constraints or prerequisites. See the pre-existing "Attach to running volume" section for an example.